tools/check.py

```diff
@@ -80,7 +89,7 @@ def _validate_no_fixme(self):
         valid = True
         for i, line in enumerate(self.markdown.splitlines()):
             if re.search("FIXME", line, re.IGNORECASE):
-                logging.error(
+                report_error("error",
```

Comment:
Alternately, we could minimize the scope of code changes by decorating `logging.error` once, rather than editing every function. Something like the following snippet (untested and may need minor tweaks- I'm still at work):

```python
import functools
def incr_error(func):
    @functools.wraps(func)
    def wrapper(*args, **kwargs):
        global NUMBER_OF_ERRORS
        NUMBER_OF_ERRORS += 1
        return f(*args, **kwargs)
    return wrapper

logging.error = incr_error(logging.error)